src/BPEssentials/Models/WarpHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BPEssentials.FileHandler;
+
+namespace BPEssentials
+{
+    public class WarpHandler : ExpandableFileHandler<WarpHandler.JsonModel>
+    {
+        public WarpHandler()
+        {
+            Name = "warp";
+            FileExtension = "json";
+            FilesDirectory = Core.Instance.Paths.WarpsFolder;
+        }
+        public class JsonModel : IExpandableFile, IExpandableFileDelayable, IExpandableFileHasPrice
+        {
+            public string Name { get; set; }
+            public bool Disabled { get; set; }
+            public int Price { get; set; }
+            public int Delay { get; set; }
+
+            public Position Position { get; set; }
+            public Rotation Rotation { get; set; }
+        }
+        public class Position
+        {
+            public float X { get; set; }
+            public float Y { get; set; }
+            public float Z { get; set; }
+            public int PlaceIndex { get; set; }
+        }
+
+        public class Rotation
+        {
+            public float X { get; set; }
+            public float Y { get; set; }
+            public float Z { get; set; }
+            public float W { get; set; }
+        }
```

Comment:
Move these types somewhere more.. public because I'm sure we're going to have to use them again someday. Maybe even make it a type in BP-CoreLib and have it include some `ToVector3()` methods which convert it?